docs/reference/configuration/raftstore-config.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Raftstore Configuration 
+summary: Learn about Raftstore configuration in TiKV.
+category: reference
+---
+
+# Raftstore Configurations
+
+Raftstore is TiKV's implementation of [Multi-raft](https://tikv.org/deep-dive/scalability/multi-raft/) to manage multiple Raft consensus groups on one node. Raftstore is comprised of two major components:
+
+- **Raftstore** component writes Raft logs into RaftDB.
+- **Apply** component resolves Raft logs and flush the data in the log into the underlying storage engine.
+
+This section introduces the following features of Raftstore and their configurations:
+
+- [Multi-thread Raftstore](#multi-thread-raftstore)
```

Comment:
How about threaded raftstore?

Reply:
Sounds fine with me. But it's in the release note already.